### PR TITLE
generate site.subdomains without periods

### PR DESCRIPTION
**Pull request by amirbey**
subdomains with "." generate preview urls that are insecure and not valid for https wildcard domains

### CODE REVIEW

File: migrations/20200930102246-update-event-created-at-to-timestamp.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ exports.up = (db, callback) => db.changeColumn('event', "createdAt", {
   type: 'timestamp',
 }, callback);
 
-exports.down = (db, callback) => db.addColumn('event', "createdAt", {
+exports.down = (db, callback) => db.changeColumn('event', "createdAt", {
```
Comment (amirbey) [inline migrations/20200930102246-update-event-created-at-to-timestamp.js:5]: good catch ... thanks

File: api/controllers/site.js [hunk 2]
```diff
@@ -163,6 +163,7 @@ module.exports = {
         res.json(siteJSON);
       })
       .catch((err) => {
+        console.error(err)
```
Comment (amirbey) [inline api/controllers/site.js]: is  this for testing?

Comment (davemcorwin) [reply api/controllers/site.js]: I caught this one already!

File: api/utils/index.js [hunk 3]
```diff
@@ -65,8 +65,31 @@ function generateS3ServiceName(owner, repository) {
   return slicedServiceName;
 }
 
+function toSubdomainPart(str) {
+  const characters = 'abcdefghijklmnopqrstuvwxyz';
+  let subdomain = str
+    // replace all invalid chars with '-'
+    .replace(/[^a-zA-Z-]+/g, '-')
```
Comment (davemcorwin) [inline api/utils/index.js:72]: allow numbers

**Review by apburnes — APPROVED** [review]

**Review by davemcorwin — APPROVED** [review]

**Pull request merged**